sbin/common/config_init.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# shellcheck disable=SC2153,SC2155
+# shellcheck disable=SC2153,SC2155,SC2030,SC2031
```

Comment:
I'd prefer not to ignore shellcheck where possible, what two rules are being broken?

Reply:
very strange.... now i removed disable on these two and linter did not complain.....